### PR TITLE
Dl Plugin

### PR DESCRIPTION
**Pull request by msaxena2**
Plugin for dL monitor C synthesis using ModelPlex. @bmmoore @yzhang90 Please review. I've rebased the branch on master. Let's rebase and merge to make the history a bit nicer.

### CODE REVIEW

File: examples/c/dl/dl_example.rvm [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
```
Comment (yzhang90) [inline examples/c/dl/dl_example.rvm]: Shall we replace this dummy example with the waterTank example?

File: plugins_logicrepository/dl/src/main/java/com/runtimeverification/rvmonitor/logicrepository/plugins/dl/DLPlugin.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.runtimeverification.rvmonitor.logicrepository.plugins.dl;
+
+import com.runtimeverification.rvmonitor.logicrepository.parser.logicrepositorysyntax.LogicRepositoryType;
+import com.runtimeverification.rvmonitor.logicrepository.plugins.LogicPlugin;
+
+public class DLPlugin extends LogicPlugin {
+    
+    /**
+     * Apply the Finite State Machine plugin to some input.
```
Comment (yzhang90) [inline plugins_logicrepository/dl/src/main/java/com/runtimeverification/rvmonitor/logicrepository/plugins/dl/DLPlugin.java]: Modify the comments. It doesn't apply anymore.

File: examples/c/dl/dl_example.rvm (hunk 1 above)
Comment (bmmoore) [reply examples/c/dl/dl_example.rvm]: Yes, that would be good.

**Review by bmmoore — APPROVED** [review]
This seems to cover everything.

**Pull request merged**